src/NovelAudioService.cpp

```diff
@@ -32,7 +32,6 @@ bool NovelAudioService::initializeAudio(){
     std::cout << "INFO: Mixer engaged!" << std::endl;
     isInitialized = true;
   }
-
   return isInitialized;
```

Comment:
@tannergooding In reference to the last review you did with calling a local that was not initialized, would this fit the bill?

Reply:
Yes. It's being explicitly set in the constructor, so looks good.